### ISSUE TITLE
ReferenceError: Map is not defined is jsts

### ISSUE THREAD

**Issue by OliveAwesomesauz** [Turfjs/turf#391]
This error happens when I require() turf:

```
ReferenceError: Map is not defined
    at /Users/oliviaw/work/purgatory/node_modules/turf/node_modules/turf-union/node_modules/jsts/dist/jsts.min.js:9:23521
    at e.hasOwnProperty.t.(anonymous function) (/Users/oliviaw/work/purgatory/node_modules/turf/node_modules/turf-union/node_modules/jsts/dist/jsts.min.js:6:69)
    at Object.<anonymous> (/Users/oliviaw/work/purgatory/node_modules/turf/node_modules/turf-union/node_modules/jsts/dist/jsts.min.js:6:161)
```

Please let me know if this issue is upstream (jsts) and I'll report it there.

**Comment by roblav96** [comment]
I fear the knowledge of what you do for work 😨  
`work/purgatory`
lol

**Comment by jenningsanderson** [comment]
@OliveAwesomesauz, I had the same error and was able to solve it by upgrading node to the latest (v6.2.1), then upgrading npm, and then re-installing (upgrading in some cases) all of the dependencies for turf.  I'm sure that's an overkill method -- but it worked for me!

**Comment by tmcw** [comment]
Fixed upstream in jsts https://github.com/bjornharrtell/jsts/commit/61d1229bc936cb33d32d212e877b70d3afb6a203 and published in v3.0.18 packages of Turf: https://github.com/Turfjs/turf/commit/fe9399a7b60fe25aff1a4ab7115d9ea94f3a9628
